### PR TITLE
Add pre commit hooks for black and isort

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
```
Comment (ch-kr) [inline .pre-commit-config.yaml:4]: ooh the version in gnomad methods is v2.3.0, should we update that?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 22.3.0 # This should be kept in sync with the version in requirements-dev.in
+    hooks:
+      - id: black
+        language_version: python3
+        files: gnomad_qc
+  - repo: https://github.com/pycqa/isort
```
Comment (ch-kr) [inline .pre-commit-config.yaml:15]: should we add pydocstyle here too?
```suggestion
  - repo: https://github.com/pycqa/pydocstyle
    rev: 6.0.0 # This should be kept in sync with the version in requirements-dev.in
    hooks:
      - id: pydocstyle
  - repo: https://github.com/pycqa/isort
```

File: requirements-dev.txt [hunk 3]
```diff
@@ -0,0 +1,79 @@
+#
+# This file is autogenerated by pip-compile with python 3.7
```
Comment (ch-kr) [inline requirements-dev.txt:2]: this says 3.7 but the pre-commit config is 3.8?

File: requirements-dev.in [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Used to create the requirements-dev.txt file
+# Note: `pip-compile requirements-dev.in` needs to be run before changes in requirements-dev.in are reflected in
+# requirements-dev.txt, which is used for GitHub Actions
+black==22.3.0 # This should be kept in sync with the version in .pre-commit-config.yaml
+isort==5.10.1 # This should be kept in sync with the version in .pre-commit-config.yaml
```
Comment (ch-kr) [inline requirements-dev.in:5]:
```suggestion
isort==5.10.1 # This should be kept in sync with the version in .pre-commit-config.yaml
pydocstyle==6.0.0 # This should be kept in sync with the version in .pre-commit-config.yaml
```

**Review by ch-kr — COMMENTED** [review]
quick question about python version and pydocstyle; also wanted to note I didn't look at all of the file changes (assumed they were black/isort updates)

**Review by ch-kr — APPROVED** [review]
approving now since I'll be OOO and my comments were small

File: .pre-commit-config.yaml (hunk 2 above)
Comment (jkgoodrich) [reply .pre-commit-config.yaml:15]: Too many errors, I'm trying to get it added now, but in a different PR so the reviewer can actually see how I resolved the errors

File: .pre-commit-config.yaml (hunk 1 above)
Comment (jkgoodrich) [reply .pre-commit-config.yaml:4]: Might as well

File: .pre-commit-config.yaml (hunk 2 above)
Comment (ch-kr) [reply .pre-commit-config.yaml:15]: oh good point, I forgot it would probably edit a bunch of scripts

File: requirements-dev.txt (hunk 3 above)
Comment (jkgoodrich) [reply requirements-dev.txt:2]: Thank you!

**Pull request merged**